whois-commons/src/test/java/net/ripe/db/whois/common/rpki/DummyRpkiDataProvider.java

```diff
@@ -13,8 +14,14 @@
 @Profile({WhoisProfile.TEST})
 public class DummyRpkiDataProvider implements RpkiDataProvider {
 
+    private List<Roa> loadedRoas = Lists.newArrayList();
+
     @Override
     public List<Roa> loadRoas() {
+        if (!loadedRoas.isEmpty()){
+            return loadedRoas;
+        }
+
         try {
             return new ObjectMapper().readValue(getClass().getResourceAsStream("/rpki/roas.json"), Roas.class).getRoas();
```

Comment:
why does reading `roas.json` not set `loadedRoas` ? The behaviour is confusing

Reply:
I can do 
            loadedRoas = new ObjectMapper().readValue(getClass().getResourceAsStream("/rpki/roas.json"), Roas.class).getRoas();
            return loadedRoas;
            
But in the test will not make any difference

Reply:
talk over zoom